### PR TITLE
Strings export 2017-03-14

### PR DESCRIPTION
**Pull request by pocmo**
Now including the error page strings and some strings for leaving the app.

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -22,6 +22,11 @@ msgctxt "action_cancel"
 msgid "Cancel"
 msgstr ""
 
+#. Label used for buttons, e.g. in dialogs
+msgctxt "action_ok"
+msgid "Ok"
```
Comment (flodolo) [inline locales/templates/app.pot]: In English is OK, not Ok.

File: locales/templates/app.pot [hunk 2]
```diff
@@ -229,3 +234,136 @@ msgctxt "firstrun_footer"
...
+
+msgctxt "external_app_prompt_no_app_title"
+msgid "No app to handle link"
+msgstr ""
+
+#, c-format
+msgctxt "external_app_prompt_no_app"
+msgid ""
+"None of the apps on your device are able to open this link. Do you want "
+"to exit %1$s and open %2$s to find a suitable app?"
```
Comment (flodolo) [inline locales/templates/app.pot:263]: Are variables swapped compared to the previous string? Can we have a comment?

File: locales/templates/app.pot [hunk 3]
```diff
@@ -229,3 +234,136 @@ msgctxt "firstrun_footer"
...
+msgstr ""
+
+msgctxt "error_hostLookup_title"
+msgid "Server not found"
+msgstr ""
+
+msgctxt "error_hostLookup_message"
+msgid ""
+"<ul>\n"
+"        <li>Check the address for typing errors such as\n"
```
Comment (flodolo) [inline locales/templates/app.pot:306]: These \n seem like an artifact of the conversion?

File: locales/templates/app.pot [hunk 4]
```diff
@@ -229,3 +234,136 @@ msgctxt "firstrun_footer"
...
+msgstr ""
+
+msgctxt "error_malformedURI_title"
+msgid "The address isn’t valid"
+msgstr ""
+
+msgctxt "error_malformedURI_message"
+msgid ""
+"<ul>\n"
+"  <li>Web addresses are usually written like "
```
Comment (flodolo) [inline locales/templates/app.pot:321]: This, for example, doesn't have the \n.

**Review by Delphine — APPROVED** [review]
Looks good to me

File: locales/templates/app.pot (hunk 1 above)
Comment (pocmo) [reply locales/templates/app.pot]: I'll update that one.

File: locales/templates/app.pot (hunk 2 above)
Comment (pocmo) [reply locales/templates/app.pot:263]: I'll add a comment.

File: locales/templates/app.pot (hunk 3 above)
Comment (pocmo) [reply locales/templates/app.pot:306]: Those strings should be a 1:1 import from Fennec's netError.dtd (minus the forms). That's how those strings look like in our Android world:
https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/res/values/strings.xml#L138

Looking at the existing string "teaser" (which also includes a line break): Those show up in our tools like Pontoon like regular line breaks, right?

Comment (flodolo) [reply locales/templates/app.pot:306]: Correct, given that teaser doesn't show up as a literal \n, I think we can keep these.

**Review by flodolo — APPROVED** [review]

**Pull request merged**